care.config.ts

```diff
@@ -143,6 +143,10 @@ const careConfig = {
   resendOtpTimeout: env.REACT_APP_RESEND_OTP_TIMEOUT
     ? parseInt(env.REACT_APP_RESEND_OTP_TIMEOUT, 10)
     : 30,
+
+  MAX_IMAGE_UPLOAD_SIZE_MB: env.REACT_APP_MAX_IMAGE_UPLOAD_SIZE_MB
+    ? parseInt(env.REACT_APP_MAX_IMAGE_UPLOAD_SIZE_MB, 10)
+    : 2,
```

Comment:
keys of care config must follow js naming conventions.

```suggestion
  imageUploadMaxSizeInMB: env.REACT_APP_MAX_IMAGE_UPLOAD_SIZE_MB
    ? parseInt(env.REACT_APP_MAX_IMAGE_UPLOAD_SIZE_MB, 10)
    : 2,
```